server/api/crud/model_monitoring/model_endpoints.py

```
@@ -547,6 +554,58 @@ def delete_model_endpoints_resources(project_name: str):
             # store not found, there is nothing to delete
             pass
 
+        # Delete model monitoring stream resources
+        self._delete_model_monitoring_stream_resources(
+            project_name=project_name,
+            db_session=db_session,
+            model_monitoring_applications=model_monitoring_applications,
+            stream_paths=stream_paths,
+        )
+
+    @staticmethod
+    def _delete_model_monitoring_stream_resources(
+        project_name: str,
+        db_session: sqlalchemy.orm.Session,
+        model_monitoring_applications: typing.Optional[list[str]] = None,
```

This type hint and default `None` value is misleading - see below

---

I don't see any change here